homeassistant/components/media_player/__init__.py

```diff
@@ -848,27 +853,29 @@ async def async_browse_media(
         """
         raise NotImplementedError()
 
+    async def _async_fetch_image(self, url, cache=True):
```

Comment:
This method should not have optional caching, that makes little sense. Instead break out the image fetching part.

Reply:
Ok, I broke out that part into `_async_retrieve_image`. I'm not sure these method names are obvious. Do you have a suggestion for more intuitive names?